project/src/components/city-card/city-card.tsx

```diff
@@ -0,0 +1,55 @@
+function CityCard():JSX.Element {
+  return (
+    <article className='cities__card place-card'>
+      <div className='place-card__mark'>
+        <span>Premium</span>
+      </div>
+      <div className='cities__image-wrapper place-card__image-wrapper'>
+        <a href='#'>
```

Comment:
а тут? чувство прекрасного тебя подвело :)
```suggestion
        <a href={'#apartment-03'}>
```